deploy/examples/subvolumegroup.yaml

```diff
@@ -2,10 +2,18 @@
 apiVersion: ceph.rook.io/v1
 kind: CephFilesystemSubVolumeGroup
 metadata:
-  name: group-a
+  name: group-a15
   namespace: rook-ceph # namespace:cluster
 spec:
   # The name of the subvolume group. If not set, the default is the name of the subvolumeGroup CR.
   # name: group-a
   # filesystemName is the metadata name of the CephFilesystem CR where the subvolume group will be created
   filesystemName: myfs
+  # reference https://docs.ceph.com/en/quincy/cephfs/fs-volumes/#pinning-subvolumes-and-subvolume-groups
+  # only one out of (export, distributed, random) can be set at a time
+  # pinning:
```

Comment:
Let's enable the setting so it's more obvious what the default is:
```
    pinning:
      distributed: 1
```
If it's commented out, it's less obvious the default.

Reply:
Updated